lib/beaker/test_suite.rb

```diff
@@ -141,7 +141,14 @@ def summarize(summary_logger)
       #@param [TestCase] test_case The {TestCase} to examine and print results for
       def print_test_result(test_case)
         test_reported = if test_case.exception
-                          "reported: #{test_case.exception.inspect}"
+                          test_file_trace = ""
```

Comment:
this block needs indent fixes. A block should only be indented 2 spaces from its parent code.